beakerx/beakerx/tabledisplay/tabledisplay.py

```diff
@@ -91,7 +93,10 @@ def convertType(self, object_type, value):
         if object_type == "float64":
             return "double"
         if object_type == "int64":
-            return "integer"
+            if value > (pow(2, 31) - 1) or value < -(pow(2, 31)):
```

Comment:
use a standard constant for this eg https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.iinfo.html
if pandas has one that's better.

Reply:
done. numpy used.